db/seeds.rb

```diff
@@ -8,27 +8,61 @@
             travesty_id: travesty.id,
             image_url: "oil_barrel.jpg")
 
+Item.create(title: "Oil Barrels",
+            description: "holds oil",
+            price: 10,
+            travesty_id: travesty.id,
+            image_url: "oil_barrel.jpg")
+
+
 Item.create(title: "6 Pack Plastic Rings",
             description: "Great for hugging animals!",
             price: 1000,
             travesty_id: travesty.id,
             image_url: "turtle_ocean.jpg")
 
+Item.create(title: "7 Pack Plastic Rings",
+            description: "Great for hugging animals!",
+            price: 1000,
+            travesty_id: travesty.id,
+            image_url: "turtle_ocean.jpg")
+
+
 Item.create(title: "HairDryer Icecap Melting Team",
             description: "Great for fun in the snow/water!",
             price: 10,
             travesty_id: travesty.id,
             image_url: "polar_icecap.jpg")
 
+Item.create(title: "HairDryer Icecap Melting Teams",
+            description: "Great for fun in the snow/water!",
+            price: 10,
+            travesty_id: travesty.id,
+            image_url: "polar_icecap.jpg")
+
+
 Item.create(title: "Black Rhino hunting license",
             travesty_id: travesty_2.id,
             description: "Mother not included.",
             price: 20000,
             image_url: "rhino_black.jpg")
 
+Item.create(title: "White Rhino hunting license",
+            travesty_id: travesty_2.id,
+            description: "Mother not included.",
+            price: 20000,
+            image_url: "rhino_black.jpg")
+
+
 
 Item.create(title: "Bowhead Whale hunting license",
             travesty_id: travesty_2.id,
             description: "Enough fat to light 10 million candles",
             price: 10000,
             image_url: "whale_hunting.jpg")
+
+Item.create(title: "Killer Whale hunting license",
+          travesty_id: travesty_2.id,
+          description: "Enough fat to light 10 million candles",
+          price: 10000,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Separate every 3 digits in the integer portion of a number with underscores(_).